src/pages/index.js

```diff
@@ -137,94 +247,395 @@ function InstallationSection() {
+            <h2 className="title">
+              <Translate
+                id="community.nextRelease.title"
+                description="Title for the next feature release section">
+                Next Feature Release
+              </Translate>
+            </h2>
             <dl>
               <dt>
-                <strong>Version:</strong> v3.16.3
+                <strong>
+                  <Translate
+                    id="community.nextRelease.versionLabel"
+                    description="Label for version">
+                    Version:
+                  </Translate>
+                </strong> v3.16.3
                 <br />
-                <strong>Date:</strong> 2024-11-12
+                <strong>
+                  <Translate
+                    id="community.nextRelease.dateLabel"
+                    description="Label for date">
+                    Date:
+                  </Translate>
+                </strong>{' '}
+                2024-11-12
                 <br />
                 <br />
-                <a href="calendar/release">Release Calendar</a>
+                <a href="calendar/release">
+                  <Translate
+                    id="community.nextRelease.calendarLink"
+                    description="Link to release calendar">
+                    Release Calendar
+                  </Translate>
+                </a>
               </dt>
             </dl>
           </section>
 
           <section className="box">
-            <h2 className="title">Events</h2>
+            <h2 className="title">
+              <Translate
+                id="community.events.title"
+                description="Title for the events section">
+                Events
+              </Translate>
+            </h2>
             <dl>
-              <dt>Upcoming Events</dt>
-              <dd><small className="placeholder"><em>Nov 10-13th 2025</em> - <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon North America</a></small></dd>
-              <dt>Past Events</dt>
-              <dd><em>Apr 1-4th 2025</em> - <a href="https://events.linuxfoundation.org/archive/2025/kubecon-cloudnativecon-europe/">KubeCon Europe 2025</a></dd>
-              <dd><em>Nov 12-15th 2024</em> - <a href="https://events.linuxfoundation.org/archive/2024/kubecon-cloudnativecon-north-america/">KubeCon North America 2024</a></dd>
-              <dd><em>May 19-22nd 2024</em> - <a href="https://events.linuxfoundation.org/archive/2024/kubecon-cloudnativecon-europe/">KubeCon Europe 2024</a></dd>
+              <dt>
+                <Translate
+                  id="community.events.upcoming"
+                  description="Label for upcoming events">
+                  Upcoming Events
+                </Translate>
+              </dt>
+              <dd>
+                <small className="placeholder">
+                  <em>
+                    2025-11-10 to 2025-11-13
```

Comment:
>2025-11-10 to 2025-11-13

For discussion: I changed Nov 10, 2025 to 2025-11-10. Do we want to use international date format throughout? It would make these docs more internationalized natively (and be one less thing that needs to be translated)